LoopKit/TherapySetting.swift

```diff
@@ -59,7 +59,7 @@ public extension TherapySetting {
         case .deliveryLimits:
             return "\(DeliveryLimits.Setting.maximumBasalRate.descriptiveText)\n\n\(DeliveryLimits.Setting.maximumBolus.descriptiveText)"
         case .insulinModel:
-            return LocalizedString("The app assumes insulin is actively working for 6 hours. You can choose from two different models for how the app measures the insulin’s peak activity.", comment: "Descriptive text for insulin model")
+            return LocalizedString("The app assumes insulin is actively working for 6 hours. You can choose from several different models for how the app measures the insulin’s peak activity.", comment: "Descriptive text for insulin model")
```

Comment:
[nit] "several" seems like an exaggeration when Tidepool Loop will ship with only two, though I understand the intention to make this text static. Maybe we just drop the quantifier and say "You can choose from different models..."

Reply:
I like it.  I wasn't thrilled with "several" TBH.  I'll drop it.